Move jBPM Performance Tests to the kie-benchmarks repository

MarianMacik opened the pull request:
Hi, @baldimir,

here are the jBPM Performance Tests. Bear in mind that they are still work in progress for 7.0 (but they are compilable and runnable) so in case there are any code smells I will focus on them after this PR is merged. Do not merge this yet please as I would like go through it on GitHub as well.

Thanks!

Marian

MarianMacik commented on pom.xml:
```diff
@@ -29,9 +29,16 @@
         <artifactId>drools-bom</artifactId>
-        <version>${drools.bom.version}</version>
+        <version>${version.org.kie}</version>
         <type>pom</type>
```
@baldimir I have question about this - should we use jbpm + drools BOM or whole kie-platform-bom?

baldimir replied:
We can leave just kie-platform-bom there. It will be better, because also OptaPlanner tests would be migrated sometime in the future eventually.

MarianMacik commented:
I have updated pom.xml with kie-parent-bom and introduced **kie.bom.version** property for its version. Now it should be OK and ready to merge. Thanks!
